src/main/java/lottogame/controller/LottoGameController.java

```diff
@@ -16,14 +19,21 @@ public LottoGameController() {
     }
 
     public void play() {
-        LottoTickets lottoTickets = getLottoTickets(new Money(InputView.inputMoney()));
-        OutputView.printLottoTickets(lottoTickets.getLottoTickets());
+        Money money = new Money(InputView.inputMoney());
+        Count manualTicketCount = new Count(InputView.inputManualTicketCount());
+        LottoGame lottoGame = new LottoGame(new LottoTicketIssueMachine(money, manualTicketCount));
+        LottoTickets lottoTickets = getLottoTickets(manualTicketCount, lottoGame);
         LottoWinningNumbers lottoWinningNumbers = getWinningNumbers();
+        Map<Rank, Integer> ranks = lottoGame.getMatchingResult(lottoTickets, lottoWinningNumbers);
+        printLottoGameResult(lottoGame.getMatchingResult(lottoTickets, lottoWinningNumbers), lottoGame.getYield(ranks));
```

Comment:
```suggestion
        Map<Rank, Integer> ranks = lottoGame.getMatchingResult(lottoTickets, lottoWinningNumbers);
        printLottoGameResult(rank, lottoGame.getYield(ranks));
```
1. 그런데 `printLottoGameResult()`에게 넘겨주기 위해 `lottoGame`에서 계속 뭘 꺼내고 있군요. 게다가 이 꺼낸 것들은 서로 굉장히 연관이 있어 보이고요..? 그렇다면?
2. `play()` 메서드 안에서 적절히 줄바꿈을 해 주세요.

Reply:
> 그런데 printLottoGameResult()에게 넘겨주기 위해 lottoGame에서 계속 뭘 꺼내고 있군요. 게다가 이 꺼낸 것들은 서로 굉장히 연관이 있어 보이고요..? 그렇다면?

`getYield` 내부에서 `getMatchingResult`를 호출하는 것으로 변경해봤어요!
또한 이렇게 하는 것이 당첨번호 비교 결과는 출력하지 않고 수익률만 출력하도록 로직이 변경되었을 때도 `play` 메서드 내부 흐름이 자연스러울 것이라고 생각되었어요!